docs/source/contributing/writing-docs.adoc

```diff
@@ -159,3 +159,10 @@ $ rake build
 ----
 
 If the build completes successfully, the site is available under *_preview/openshift-origin/latest/welcome/index.html_*.
+
```

Comment:
@robin-owen Please check the documentation here. Let me know your comments/suggestions.